File: src/main/java/gq/noxiuam/bitches/exception/NoBitchesException.java
package gq.noxiuam.bitches.exception;

public class NoBitchesException extends RuntimeException {

    private final int amount;

    public NoBitchesException(int amount) {
        this.amount = amount;
    }

    public String getMessage() {
        return "You got " + this.amount + " and was unable to use the program, oops!" + (this.amount == 0 ? "No bitches?" : "");
    }

}

File: src/main/kotlin/gq/noxiuam/bitches/exception/NoBitchesException.kt
package gq.noxiuam.bitches.exception

class NoBitchesException(private val amount: Int) : RuntimeException() {
    override val message: String
        get() = "You got " + amount + " and was unable to use the program, oops!" + if (amount == 0) "No bitches?" else ""
}